packages/node-cli/messages.json

```diff
@@ -122,6 +126,32 @@
 	"COMMAND_LOCAL_RESOURCE_NOT_FOUND": "The process failed. The {0} file was not found in the project.",
 	"COMMAND_LOCAL_INVALID_XML": "Invalid xml {0}",
 
+	"COMMAND_MANAGE_ACCOUNT_ERROR_CREDENTIALS_EMPTY": "There is no authentication ID (authID) configured for this account-role combination. Run \"suitecloud account:setup\" to set up a NetSuite account to use with SuiteCloud CLI for Node.js.",
+	"COMMAND_MANAGE_ACCOUNT_ERROR_TAG": "The Authentication Tag/Label provided, is not valid, please review the data.",
+	"COMMAND_MANAGE_ACCOUNT_ERROR_UNKNOWN_ACTION": "Unknown selected account:manageauth action",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_ACCOUNT_INFO_ACCOUNT_ID": "Account ID: {0}",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_ACCOUNT_INFO_ACCOUNT_NAME": "Account Name: {0}",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_ACCOUNT_INFO_ACCOUNT_TYPE": "Account Type: {0}",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_ACCOUNT_INFO_AUTHID": "Authentication ID (authID): {0}",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_ACCOUNT_INFO_DOMAIN": "Domain: {0}",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_ACCOUNT_INFO_ROLE": "Role: {0}",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_CANCEL": "The manageauth process has been canceled.",
+	"COMMAND_MANAGE_ACCOUNT_MESSAGES_INFO": "Loading \"{0}\" authentication ID (authID) information:",
```

Comment:
Are we going to change authentication ID with authentication alias?

Reply:
Not in this story, but in a following one.